Vagrantfile

```diff
@@ -244,6 +244,18 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
                     end
                     formula_path
                 }
+                # write the minion file
+                minion_cfg = YAML.load_file("scripts/salt/minion.template")
+                more_base_paths = formula_paths.map{|p| "/vagrant/" + p + "/"}
+                minion_cfg['file_roots']['base'].insert(1, *more_base_paths)
```

Comment:
huh! in Python you can assign to a slice operation: `x = [1,2,3]; x[1:1] = ['a', 'b']; => [1,'a','b',2,3]`